vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/CsvFileDataSource.java

```diff
@@ -22,103 +22,53 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.util.Arrays;
 
-import org.apache.commons.lang3.Validate;
 import org.eclipse.jdt.annotation.NonNull;
-import org.semanticweb.vlog4j.core.model.api.TermType;
-import org.semanticweb.vlog4j.core.reasoner.DataSource;
 
 /**
- * A CsvFileDataSource stores fact terms (tuples) as lines in a ".csv" format
- * file, each column being a predicate argument.
- * 
+ * A {@code CsvFileDataSource} stores facts as tuples inside a file of the
+ * extension {@code .csv}. One file can only include facts of a single
+ * predicate. The required format looks like this
```

Comment:
Thank you for being to eager to review my code. I wasn't finished with all my fixes, however, I just like to push all my commits onto the repository as quickly as possible. I'm not too familiar with the review procedure on GitHub -- whether new reviews are automatically requested after every push, or whether the pull requester has to request an updated review... In case reviews are automatically requested, I will take that into account when I push.

Reply:
Reviews are not automatically requested after each push. However, since this is a rework after a review, I was not sure you can request a new review, so I commented on the fresh changes. I can wait for a notification from you, when you have finished the rework after review, to review again.